CHANGELOG.md

```diff
@@ -87,4 +87,3 @@
 * Unidentified Item Icons ([#271](https://github.com/Wynntils/Artemis/issues/271)) ([3d73347](https://github.com/Wynntils/Artemis/commit/3d73347b00718981381f3670a32ae09d35b5528f))
 
 ## 0.0.0 (2021-12-09)
-
```

Comment:
Is this a spurious change triggered by the new build publishing tooling?

Reply:
yeah + spotless, might be a one off though

Reply:
Ok, we'll need to keep an eye on this so it does not start ping-ponging.